bundles/org.openhab.binding.evcc/README.md

```diff
@@ -68,17 +126,175 @@ Bridge evcc:server:demo-server "Demo" [scheme="https", host="demo.evcc.io", port
+Number:EnergyPrice Evcc_Site_Tariff_Feed_In "Feed in price per kWh [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Tariff_Feed_In" }
+Number:EnergyPrice Evcc_Site_Tariff_Grid "Grid Price [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Tariff_Grid" }
+Number:EnergyPrice Evcc_Site_Tariff_Price_Home "Home Price [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Tariff_Price_Home" }
+Number:EnergyPrice Evcc_Site_Tariff_Price_Loadpoints "Loadpoint price per kWh [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Tariff_Price_Loadpoints" }
+Number:Power Evcc_Site_Tariff_Solar "Solar Tariff [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Tariff_Solar" }
+String Evcc_Site_Site_Title "Site Title [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Site_Title" }
+String Evcc_Site_Version "Version [%s]" { channel="evcc:battery:demo-server:demo-battery:Site_Version" }
+Number:EmissionIntensity Evcc_Avg_Co2_Type "Average CO2 [%s]" { channel="evcc:battery:demo-server:demo-battery:Avg_Co2_Type" }
+Number:EnergyPrice Evcc_Avg_Price_Type "Average Price [%s]" { channel="evcc:battery:demo-server:demo-battery:Avg_Price_Type" }
+Number:Energy Evcc_Charged_Energy_Type "Charged Energy [%s]" { channel="evcc:battery:demo-server:demo-battery:Charged_Energy_Type" }
+Number:Dimensionless Evcc_Solar_Percentage_Type "Solar Percentage [%s]" { channel="evcc:battery:demo-server:demo-battery:Solar_Percentage_Type" }
+Number:Energy Evcc_Vehicle_Capacity "Vehicle Capacity [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Capacity" }
+String Evcc_Vehicle_Icon "Vehicle Icon [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Icon" }
+Number:Dimensionless Evcc_Vehicle_Limit_Soc "Vehicle Limit SoC [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Limit_Soc" }
+Number:Dimensionless Evcc_Vehicle_Min_Soc "Vehicle Min SoC [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Min_Soc" }
+Number:ElectricCurrent Evcc_Vehicle_Max_Current "Vehicle Max Current [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Max_Current" }
+Number:ElectricCurrent Evcc_Vehicle_Min_Current "Vehicle Min Current [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Min_Current" }
+Number:Time Evcc_Vehicle_Plan_Precondition "Plan Precondition Duration [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Plan_Precondition" }
+Number:Dimensionless Evcc_Vehicle_Plan_Soc "Planned SoC [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Plan_Soc" }
+DateTime Evcc_Vehicle_Plan_Time "Plan Datetime [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Plan_Time" }
+String Evcc_Vehicle_Title "Vehicle Title [%s]" { channel="evcc:battery:demo-server:demo-battery:Vehicle_Title" }
 ```
 
+## Migration / Update
+
+Renamed property siteTitle -> site-title
+Renamed property id -> vehicle-id and renamed parameter id -> vehicle-id, you may need to parametrize you thing.
+
```

Comment:
Couyld you elaborate on this? It is not common to list releae notes / changelog in the documentation.